### ISSUE TITLE
Accuracy Drop on the Validation data of Nuscene Mini

### ISSUE THREAD

**Issue by abhigoku10** [hustvl/Senna#26]
@xinggangw @wondervictor @ifzhang @LegendBC @rb93dett  Hi all thanks for sharing ur work , i have evaluated the model on the Nuscences Mini dataset and the accuracy seems to be low ie around 40% is there any reason behind this ?? I have generated few results viz and checked the output which is shared below can u let me know if anything is going wrong with our testing 
![ezgif com-animated-gif-maker](https://github.com/user-attachments/assets/3801e026-51b3-4cbf-8e51-91b2573fec8d)

Thanks in advance

**Comment by HAMA-DL-dev** [comment]
I also sincerely appreciate the code sharing for reproduction. 

As @abhigoku10  mentioned, I have observed a decline in both qualitative and quantitative results (F1 score) on the nuscenes_mini dataset. 

I was wondering if I might have made any mistakes or if the authors have encountered a similar issue before.

Best regards,

```bash
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1171/1171 [00:57<00:00, 20.20it/s]


=========== F1 Score ===========


KEEP: 0.962406015037594
ACCELERATE: 0
DECELERATE: 0
STOP: 0
RIGHT_TURN: 0
LEFT_TURN: 0.07142857142857144
STRAIGHT: 0.6185567010309277


================================



Total Number: 69


Correct Number: 27


------------------------------


Planning Accuracy: 39.13%


------------------------------

KEEP_RIGHT_TURN: num: 13, correct num: 0, 0.00%
KEEP_LEFT_TURN: num: 2, correct num: 1, 50.00%
KEEP_STRAIGHT: num: 50, correct num: 26, 52.00%
DECELERATE_STRAIGHT: num: 4, correct num: 0, 0.00%

Eval results saved to /home/mmc-server4/Server/Users/hayeon/Senna/output/eval/result/pred_result.json
```

**Comment by Cynthia11281128** [comment]
> I also sincerely appreciate the code sharing for reproduction.
> 
> As [@abhigoku10](https://github.com/abhigoku10) mentioned, I have observed a decline in both qualitative and quantitative results (F1 score) on the nuscenes_mini dataset.
> 
> I was wondering if I might have made any mistakes or if the authors have encountered a similar issue before.
> 
> Best regards,
> 
> 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1171/1171 [00:57<00:00, 20.20it/s]
> 
> 
> =========== F1 Score ===========
> 
> 
> KEEP: 0.962406015037594
> ACCELERATE: 0
> DECELERATE: 0
> STOP: 0
> RIGHT_TURN: 0
> LEFT_TURN: 0.07142857142857144
> STRAIGHT: 0.6185567010309277
> 
> 
> ================================
> 
> 
> 
> Total Number: 69
> 
> 
> Correct Number: 27
> 
> 
> ------------------------------
> 
> 
> Planning Accuracy: 39.13%
> 
> 
> ------------------------------
> 
> KEEP_RIGHT_TURN: num: 13, correct num: 0, 0.00%
> KEEP_LEFT_TURN: num: 2, correct num: 1, 50.00%
> KEEP_STRAIGHT: num: 50, correct num: 26, 52.00%
> DECELERATE_STRAIGHT: num: 4, correct num: 0, 0.00%
> 
> Eval results saved to /home/mmc-server4/Server/Users/hayeon/Senna/output/eval/result/pred_result.json

I got the same results with nuscenes_mini

**Comment by dagongji10** [comment]
“The model weights we provide are pretrained on the DriveX dataset for scene understanding to facilitate fine-tuning and transfer to other specific datasets, it is recommended to fine-tune the model on the generated nuScenes data to obtain the best results.” From rb93dett.  https://github.com/hustvl/Senna/issues/11#issuecomment-2558848826

**Comment by hypez66** [comment]
Can you share me the converted data or guide me how to boost the converting process? I will appreciate it very much, thanks!
